deploy/kubeturbo/values.yaml

```diff
@@ -19,9 +19,10 @@ serverMeta:
   version: TURBONOMIC_SERVER_VERSION
   turboServer: https://Turbo_server_URL
 
-# Turbo server api user and password
+# Turbo server api user and password stored in a secret or optionally specified as username and password
 # The opsManagerUserName requires Turbo administrator role
 restAPIConfig:
+  turbonomicCredentials: "turbonomic-credentials"
```

Comment:
Suggest to name the var such that users could clearly tell this is a secret name, e.g. `turbonomicCredentialsSecretName`.

To be consistent, I'd also change the following two vars to something like:
- `turbonomicUserName`
- `turbonomicPlaintextPassword`

Reply:
thanks @chlam4 turbonomicCredentialsSecretName makes sense, I changed it
the other two values, I cannot change, otherwise we break backward compability
but we should prefer using the secret, so eventually they can be obsoleted